src/plugins/intel_gpu/src/plugin/ops/broadcast.cpp

```diff
@@ -50,23 +50,6 @@ static void CreateCommonBroadcastOp(Program& p, const std::shared_ptr<ngraph::No
         if (axis_mapping.empty()) {
```

Comment:
Is it still needed?

Reply:
Isn't this https://github.com/openvinotoolkit/openvino/pull/14264/files#diff-2dd42824bd12930f0ff25212e0cd788f6b2e22321233de74535c009caf592517R59 doing same?

Reply:
For example, the func_test
"smoke_TestNumpyBroadcast2D/BroadcastLayerTest.CompareWithRefs/targetShape=(2.3.6)_axesMapping=()_mode=numpy_inShape=(3.1)_inNPrec=I32_trgDev=GPU" fails without this code.
In case of target shape < 4 dims, input shape should align right-side before reshape to 4 dims.

[Shape flow]

With this code:
inshape(3,1),target(2,3,6)
inshape(3,1),target(2,3,6) to inshape(1,3,1),target(2,3,6) after right align at CreateCommonBroadcastOp
inshape(1,3,1),target(2,3,6) to inshape(1,3,1,1),target(2,3,6) after reshape
inshape(1,3,1,1),target(2,3,6,1) to inshape(1,3,1,1),target(2,3,6,1) after extending input dim at get_kernel_params
Broadcast OK

Without this code:
inshape(3,1),target(2,3,6)
inshape(3,1),target(2,3,6) to inshape(3,1,1,1),target(2,3,6) after reshape
inshape(3,1),target(2,3,6,1) to inshape(1,1,3,1),target(2,3,6,1) after extending input dim at get_kernel_params
Broadcast Fail

Reply:
Without this code:
target shape become (2,3,6,1) at broadcast_inst::calc_output_layout